samples/README.md

```diff
@@ -0,0 +1,18 @@
+## DeepWorks samples
+
+Before using scripts setup enviroment:
+
+* Build deepworks with samples:
+```bash
+mkdir build
+cd build
+cmake <path-to-deepworks-root> -DBUILD_SAMPLES=ON -DDOWNLOAD_DATA=ON
+make -j8
```

Comment:
I think there is no need to duplicate the build process in samples/README.md.

Reply:
Done